src/controller/products.controller.ts

```diff
@@ -44,9 +44,31 @@ export default function (
 
           const isLifetimeUser = lifetime ?? false;
 
-          const antivirusTier = await tiersService.getProductsTier(customerId, isLifetimeUser);
+          const userSubscriptions = await paymentService.getActiveSubscriptions(customerId);
```

Comment:
To speed up this check for the free users (the big part of the client base) I would check first against our db, if possible and just after, to Stripe, in that order